docker-compose.yml

```diff
@@ -20,42 +20,13 @@ services:
         ALGOD_PORT: "4001"
         KMD_PORT: "4002"
         CDT_PORT: "9392"
+        DROP_FOLLOWER: "${INDEXER_DISABLED}"
     ports:
       - ${ALGOD_PORT:-4001}:4001
       - ${KMD_PORT:-4002}:4002
       - ${CDT_PORT:-9392}:9392
+    networks:
```

Comment:
per convo with @winder probly don't need `follower` network, and can get away with `algod:4003` from **indexer**